[ERROR] Unable to load sqlite3 module when starting thelounge

[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]


<!--- Provide a general summary of the bug in the Title above -->

------------------------------

## Expected Behavior
The container should run without any sqlite3 error messages

## Current Behavior
During startup of the container, the module for sqlite3 cannot load

## Steps to Reproduce
1. Use docker-compose.yaml provided and start up container on a machine similar to: Linux 5.15.0-1021-oracle #27-Ubuntu SMP aarch64 GNU/Linux

## Environment
**OS:** Ubuntu 22.04
**CPU architecture:**/arm64
**Docker installed via official instructions https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository:**


## Command used to create docker container (run/create/compose/screenshot)
```
version: '3.8'
services:
  thelounge:
    image: ghcr.io/linuxserver/thelounge
    container_name: thelounge
    environment:
      - TZ=Europe/Berlin
    volumes:
      - /srv/Config/Thelounge/:/config
    restart: unless-stopped
 ```

## Docker logs
```
[custom-init] No custom services found, skipping...
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service 00-legacy: starting
s6-rc: info: service 00-legacy successfully started
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
cont-init: info: running /etc/cont-init.d/01-envfile
cont-init: info: /etc/cont-init.d/01-envfile exited 0
cont-init: info: running /etc/cont-init.d/01-migrations
[migrations] started
[migrations] no migrations found
cont-init: info: /etc/cont-init.d/01-migrations exited 0
cont-init: info: running /etc/cont-init.d/10-adduser
usermod: no changes

-------------------------------------
          _         ()
         | |  ___   _    __
         | | / __| | |  /  \
         | | \__ \ | | | () |
         |_| |___/ |_|  \__/


Brought to you by linuxserver.io
-------------------------------------

To support LSIO projects visit:
https://www.linuxserver.io/donate/
-------------------------------------
GID/UID
-------------------------------------

User uid:    911
User gid:    911
-------------------------------------

cont-init: info: /etc/cont-init.d/10-adduser exited 0
cont-init: info: running /etc/cont-init.d/30-config
cont-init: info: /etc/cont-init.d/30-config exited 0
cont-init: info: running /etc/cont-init.d/99-custom-files
[custom-init] No custom files found, skipping...
cont-init: info: /etc/cont-init.d/99-custom-files exited 0
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service init-mods: starting
s6-rc: info: service init-mods successfully started
s6-rc: info: service init-mods-package-install: starting
s6-rc: info: service init-mods-package-install successfully started
s6-rc: info: service init-mods-end: starting
s6-rc: info: service init-mods-end successfully started
s6-rc: info: service init-services: starting
s6-rc: info: service init-services successfully started
s6-rc: info: service legacy-services: starting
services-up: info: copying legacy longrun thelounge (no readiness notification)
s6-rc: info: service legacy-services successfully started
s6-rc: info: service 99-ci-service-check: starting
[ls.io-init] done.
s6-rc: info: service 99-ci-service-check successfully started
2022-10-24 23:28:00 [INFO] Configuration file created at /config/config.js.
2022-10-24 23:28:00 [ERROR] Unable to load sqlite3 module. See https://github.com/mapbox/node-sqlite3/wiki/Binaries
2022-10-24 23:28:00 [INFO] The Lounge v4.3.1 (Node.js 16.17.1 on linux arm64)
2022-10-24 23:28:00 [INFO] Configuration file: /config/config.js
2022-10-24 23:28:00 [INFO] Available at http://[::]:9000/ in public mode
2022-10-24 23:28:00 [INFO] New VAPID key pair has been generated for use with push subscription.
s6-rc: info: service 99-ci-service-check: stopping
s6-rc: info: service 99-ci-service-check successfully stopped
s6-rc: info: service legacy-services: stopping
2022-10-24 23:32:27 [INFO] Exiting...
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service init-services: stopping
s6-rc: info: service 00-legacy: stopping
s6-rc: info: service init-services successfully stopped
s6-rc: info: service init-mods-end: stopping
s6-rc: info: service 00-legacy successfully stopped
s6-rc: info: service init-mods-end successfully stopped
s6-rc: info: service init-mods-package-install: stopping
s6-rc: info: service init-mods-package-install successfully stopped
s6-rc: info: service init-mods: stopping
s6-rc: info: service init-mods successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped

```

--- a/Dockerfile
+++ b/Dockerfile
@@ -21,6 +21,7 @@ RUN \
     curl \
     jq \
     yarn && \
+  ln -s /usr/bin/python3 /usr/bin/python && \
   echo "**** install the lounge irc ****" && \
   if [ -z ${THELOUNGE_VERSION+x} ]; then \
     THELOUNGE_VERSION=$(curl -sX GET "https://api.github.com/repos/thelounge/thelounge/releases/latest" | jq -r '. | .tag_name'); \

--- a/Dockerfile.aarch64
+++ b/Dockerfile.aarch64
@@ -21,6 +21,7 @@ RUN \
     curl \
     jq \
     yarn && \
+  ln -s /usr/bin/python3 /usr/bin/python && \
   echo "**** install the lounge irc ****" && \
   if [ -z ${THELOUNGE_VERSION+x} ]; then \
     THELOUNGE_VERSION=$(curl -sX GET "https://api.github.com/repos/thelounge/thelounge/releases/latest" | jq -r '. | .tag_name'); \

--- a/Dockerfile.armhf
+++ b/Dockerfile.armhf
@@ -21,6 +21,7 @@ RUN \
     curl \
     jq \
     yarn && \
+  ln -s /usr/bin/python3 /usr/bin/python && \
   echo "**** install the lounge irc ****" && \
   if [ -z ${THELOUNGE_VERSION+x} ]; then \
     THELOUNGE_VERSION=$(curl -sX GET "https://api.github.com/repos/thelounge/thelounge/releases/latest" | jq -r '. | .tag_name'); \

--- a/README.md
+++ b/README.md
@@ -255,6 +255,7 @@ Once registered you can define the dockerfile to use with `-f Dockerfile.aarch64
 
 ## Versions
 
+* **24.10.22:** - Fix sqlite3 build.
 * **12.04.22:** - Install from source using yarn.
 * **11.04.22:** - Rebasing to alpine 3.15 and switching from python2-dev to python3-dev for building node sqlite on arm.
 * **23.01.21:** - Rebasing to alpine 3.13.

--- a/readme-vars.yml
+++ b/readme-vars.yml
@@ -65,6 +65,7 @@ app_setup_block: |
 # changelog
 
 changelogs:
+  - { date: "24.10.22:", desc: "Fix sqlite3 build." }
   - { date: "12.04.22:", desc: "Install from source using yarn." }
   - { date: "11.04.22:", desc: "Rebasing to alpine 3.15 and switching from python2-dev to python3-dev for building node sqlite on arm." }
   - { date: "23.01.21:", desc: "Rebasing to alpine 3.13." }

